src/components/ConnectionsListProfiles.tsx

```diff
@@ -48,9 +58,9 @@ const ConnectionsListProfiles = ({
           key={userProfile.fromId}
         >
           <UserAvatar
-            icon={profileIcon(userProfile)}
+            icon={profiles[userProfile.fromId]?.icon?.[0]?.href}
```

Comment:
Are we not going to fallback to blockies?

Reply:
We fall back to blockies in the UserAvatar if this value is undefined.